x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/hooks.test.tsx

```diff
@@ -9,8 +9,8 @@ const mockAction = jest.fn();
 
 let mockSubcription: (state: object) => void;
```

Comment:
In this file, I just replaced `foo` and `bar` with values that are actually present in state and actions.